.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: "build-test"
+on:
+  push:
+    branches:
+      - net203
+  pull_request:
+    branches:
+      - net203
+
+jobs:
+  test-backend:
+    runs-on: ${{ matrix.image }}
+    strategy:
+      matrix:
+        image: [windows-latest]
+    env:
+      NUGET_PACKAGES: ${{ github.workspace }}/.github/nuget-packages
+      GRADLE_USER_HOME: ${{ github.workspace }}/.github/gradle
+    steps:
+      - uses: actions/checkout@v2
+
+      # Prepare
+      - name: Gradle JVM Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: build/gradle-jvm
+          key: ${{ runner.os }}.gradle-jvm.${{ hashFiles('rider-fsharp/gradle*') }}
+
+      - name: Gradle Wrapper Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: ${{ env.GRADLE_USER_HOME }}/wrapper
+          key: ${{ runner.os }}.gradle-wrapper.${{ hashFiles('rider-fsharp/gradle/**') }}
+
+      - name: NuGet Cache
```

Comment:
Looking at the [example run](https://github.com/DedSec256/fsharp-support/pull/3/checks), it doesn't seem to work (if I understand the log correctly):
```
Cache not found for input keys: Windows.nuget.457abc6dcb0d1d36ebcee59dbef4d2a5abd8dae0d906beaab873caaf8d23ecf9+da217cfbad83434fb9208c23a2bf3aa7e58bdf8b88ba1c399320075ac49c0e8f.
```

Reply:
When key doesn't match an existing cache, it's called a cache miss, and a new cache is created if the job completes successfully

Reply:
Please check if the cache works at all.

In my experiments, the overall cache volume required for the Rider plugin was too big (more than 2 GiB), so the resulting cache was being destroyed momentarily after creation.

Reply:
I tried setting up cache keys for the Unity plugin ([JetBrains/resharper-unity#1683](https://github.com/JetBrains/resharper-unity/pull/1683)) and eventually closed it without merging, because it didn't make any difference to performance - the workflow was essentially downloading zip files from a cache, or zip files from maven/gradle/whatever. So added configuration for no performance gain.

Reply:
For Azure DevOps IntelliJ (where we're unable to use build system caches due to Azure being bad), there's a very big difference.

In my experience, about 30% of all build failures are from Gradle not being able to download something (i.e. due to network errors). Utilizing the caches would nullify this kind of errors, and improve the overall CI success rate, which couldn't be a bad thing.

So, I would recommend to use cache if it's possible.

Reply:
>  a new cache is created if the job completes successfully
OK, I see, thanks. For some reason I expected it to create the cache anyway.

Reply:
@DedSec256 Have you had a chance to make sure this cache is working as expected?

Reply:
@auduchinok, yes (:
![image](https://user-images.githubusercontent.com/26364714/97202734-0da75b80-17c5-11eb-9223-42938099dc19.png)
But I removed the cache for the Gradle JVM since it worked strangely